core/tests/puppeteer/lighthouse_setup.js

```diff
@@ -258,11 +258,11 @@ const getSkillEditorUrl = async function(browser, page) {
     await page.type(skillDescriptionField, 'Skill Description here');
     await page.click(skillOpenConceptCard);
     await page.waitForSelector(skillReviewMaterialInput, {visible: true});
-    await page.waitForTimeout(5000);
+    await page.waitForTimeout(10000);
```

Comment:
Can we wait for a change in the DOM (like we do in the e2e tests) instead of for a timeout? I think that might make these tests more robust

Reply:
It doesn't seem to work on puppeteer, I think this fix should be merged to reduce flakes.

Reply:
To summarize our offline discussion, it looks like this is happening because of #12426. We might be able to handle this on the test side by waiting for the RTE toolbar to show up. Could you try that @jimbyo? I think it should be an easy fix. We just have to:

1. Use developer tools to inspect the HTML of an RTE. Find an HTML class we can use to identify the toolbar
2. Add a waitFor for that toolbar to appear in place of the timeout

Then once the fix for #12426 merges, we can undo these changes. I've reached out to EricZLou about when the fix can be merged, but in the meantime, let's keep working on a fix that we can put in the test code